modules/post/linux/gather/tor_hiddenservices.rb

```diff
@@ -60,6 +60,8 @@ def get_host
   end
 
   def find_torrc
+    fail_with(Failure::BadConfig, "'locate' command does not exist") unless command_exists?('locate')
+
     config = cmd_exec("locate 'torrc' | grep -v 'torrc.5.gz'").split("\n")
```

Comment:
Should `plocate` be used/checked as a fallback?

Reply:
Maybe. The whole logic should be rewritten.